libcxx/include/format

```diff
@@ -223,6 +223,8 @@ namespace std {
 
 #if !defined(_LIBCPP_REMOVE_TRANSITIVE_INCLUDES) && _LIBCPP_STD_VER <= 20
 #  include <locale>
+#  include <queue>
+#  include <stack>
```

Comment:
I don't know why the inclusions are wanted here... I think `<format>` should be unrelated to `<stack>` and `<queue>` when `_LIBCPP_STD_VER <= 20`. The inclusion of `<stack>` and `<queue>` was introduced in https://github.com/llvm/llvm-project/commit/04d4f4b3d4e4fd608a3bc3fe387006435f04b61d (https://reviews.llvm.org/D141290) which just implemented some parts of [P2286R8](https://wg21.link/p2286r8) (a C++23 paper).

Reply:
https://libcxx.llvm.org/DesignDocs/HeaderRemovalPolicy.html